### ISSUE TITLE
Consider editing repository setings to remove "Packages" section

### ISSUE THREAD

**Issue by matkoniecz** [nayafia/lemonade-stand#131]
"Packages No packages published" is displayed right now, fortunately this pointless section can be removed.

Edit repo page config to remove it (cog next to the description).

I am not making a PR as it is defined in proprietary github settings, not in a git repository - and I have no rights to modify repo settings.

Removing releases section in case of this repo also may be useful.

![Peek 2020-10-25 09-10](https://user-images.githubusercontent.com/899988/97102018-3a168700-16a2-11eb-8c18-7bd9a88922eb.gif)

**Comment by D15A-Jai-61** [comment]
I don't think that is necessary @matkoniecz , although that does make the page look cleaner and more "apt". But if @nayafia , you do end up removing the packages section, you might as well remove the releases section as even that is unnecessary.
